modules/openapi-generator-cli/src/test/java/org/openapitools/codegen/cmd/GenerateTest.java

```
@@ -17,536 +17,413 @@
-        setupAndRunGenericTest("--group-id", value);
-
-        new FullVerifications() {
-            {
-                configurator.setGroupId(value);
-                times = 1;
-            }
-        };
+    public void testPackageName() {
+        final String value = "io.foo.bar.baz";
+        setupAndRunGenericTest("--package-name", value);
+        verify(configurator).setPackageName(value);
     }
 
     @Test
-    public void testArtifactId() throws Exception {
-        final String value = "awesome-api";
-        setupAndRunGenericTest("--artifact-id", value);
+    public void testRequiredArgs_LongArgs() {
+        setupAndRunTest("--input-spec", "src/test/resources/swagger.yaml", "--generator-name", "java", "--output",
+                "src/main/java", false, null);
 
-        new FullVerifications() {
-            {
-                configurator.setArtifactId(value);
-                times = 1;
-            }
-        };
+        // on top of those in setupAndRunTest:
+        verify(configurator).toClientOptInput();
+        verify(configurator).toContext();
+
+        verifyNoMoreInteractions(configurator);
     }
 
     @Test
-    public void testArtifactVersion() throws Exception {
-        final String value = "1.2.3";
-        setupAndRunGenericTest("--artifact-version", value);
+    public void testRequiredArgs_ShortArgs() {
+        setupAndRunTest("-i", "src/test/resources/swagger.yaml", "-g", "java", "-o", "src/main/java", false, null, "-p", "foo=bar");
+
+        verify(configurator).addAdditionalProperty("foo", "bar");
+        verify(configurator).toClientOptInput();
+        verify(configurator).toContext();
 
-        new FullVerifications() {
-            {
-                configurator.setArtifactVersion(value);
-                times = 1;
-            }
-        };
+        verifyNoMoreInteractions(configurator);
     }
 
     @Test
-    public void testLibrary() throws Exception {
-        final String value = "library1";
```

`library1` here should have resulted in an exception, but jmockit swallowed it.